public/components/event_analytics/explorer/events_views/doc_flyout.tsx

```diff
@@ -124,13 +124,13 @@ export const DocFlyout = ({
   const flyoutBody = (
     <EuiFlyoutBody>
       <div className="obsExplorer">
-        {populateDataGrid(
+        {/* {populateDataGrid(
           explorerFields,
           getHeaders(explorerFields.queriedFields, DEFAULT_COLUMNS.slice(1), true),
           <tr className="osdDocTable__row">{memorizedTds}</tr>,
           getHeaders(explorerFields.selectedFields, DEFAULT_COLUMNS.slice(1), true),
           <tr className="osdDocTable__row">{memorizedTds}</tr>
-        )}
+        )} */}
```

Comment:
Do you want to remove this commented piece?

Reply:
Will do